variables.tf

```diff
@@ -40,6 +40,6 @@ variable "role_description" {
 
 variable "role_name" {
   type        = string
-  description = "The name to assign the IAM role that allows read-only access to the certificate in the specified S3 bucket.  Note that the \"%s\" in this value will get replaced with the hostname variable.  If there are no instances of \"%s\" present in this value, no replacement will be made and the value will be used as is.  Including more than one instance of \"%s\" in this value will result in a Terraform error, so don't do that."
+  description = "The name to assign the IAM role that allows read-only access to the certificate in the specified S3 bucket.  Note that the \"%s\" in this value will get replaced with the hostname variable.  If there are no instances of \"%s\" present in this value, no replacement will be made and the value will be used as is.  Including more than one instance of \"%s\" in this value will result in a Terraform error, so don't do that.  If the role name (after hostname replacement) is longer than the current AWS limit of 64 characters, the role name will be truncated to the first 64 characters."
```

Comment:
Should this reference that the role name is truncated to the first 64 characters either after host name replacement (which is mentioned) *or* if used as-is? Basically, no matter what only up to the first 64 characters of the the final role name are used.

Reply:
Agreed.  I tried to write it like that, but it could be clearer.  What do think about this?

> If the role name is longer than the current AWS limit of 64 characters (either as-is or after hostname replacement), the role name will be truncated to the first 64 characters.

Reply:
Succinct and to the point, I'm a fan.

Reply:
Updated in https://github.com/cisagov/cert-read-role-tf-module/pull/43/commits/af31d790e58fc88950c410a6507375f02376a2c2.